### ISSUE TITLE
Feature request: geo intent on location click

### ISSUE THREAD

**Issue by leCradle** [telegram-s/telegram#28]
At the moment the app starts its own map view when a user clicks on a location message. This is not very useful. How about sending a geo intent that users can use other maps apps (like Google maps/navigation or whatever)?
The own map view could be part of the intent receivers list, for example.

**Comment by ex3ndr** [comment]
Geo Intent is not consistent, may be i'll add menu item for opening location in external apps.

**Comment by leCradle** [comment]
Yes it isn't and it really sucks. We're using geo intents for our app too. To manage the inconsistencies, we wrote an own activity which lists the target apps and starts different geo intent types for different apps.
You can find our implementation here:
https://github.com/jonatkins/ingress-intel-total-conversion/tree/master/mobile/src/com/cradle/iitc_mobile/share
and the intent generator here:
https://github.com/jonatkins/ingress-intel-total-conversion/blob/master/mobile/src/com/cradle/iitc_mobile/share/IntentGenerator.java
The only app I know which is able to handle all different geo intent types (listed here: http://developer.android.com/guide/components/intents-common.html#Maps) is Google Maps [and of course, our own app ;)]. In our implementation, gmaps is startet with the geo:0,0?q=lat,lng(label) intent, because labeled markers are cool. All other apps are started with the default geo:latitude,longitude intent.

**Comment by ex3ndr** [comment]
Thank you for response!
Fortunately i have implemented already any kind of custom intent handling with custom dialog (that looks like intent chooser in G+ apps). I'll implement this soon.
